components/tidb_query/Cargo.toml

```diff
@@ -17,13 +29,13 @@ chrono-tz = "0.5.1"
 codec = { path = "../codec" }
 derive_more = "0.15.0"
 failure = "0.1"
-flate2 = { version = "1.0", features = ["zlib"], default-features = false }
+flate2 = { version = "1.0", default-features = false, features = ["zlib"] }
 hex = "0.3"
 indexmap = { version = "1.0", features = ["serde-1"] }
 kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
 lazy_static = "1.3"
 match_template = { path = "../match_template" }
-nom = "5.1.0"
```

Comment:
Why not using default features?

Reply:
The default features of `nom5` is `default = ["std", "lexical"]`. For `tikv`, the `lexical` feature is not used, but it introduces many redundant dependencies, so I remove it.

Reply:
Great job!